src/editor/map/map_context.cpp

```diff
@@ -318,7 +318,8 @@ void map_context::load_scenario(const game_config_view& game_config)
 	config scenario;
 
 	try {
-		read(scenario, *(preprocess_file(filename_)));
+		//todo: maybe make filename_ a wml_path ?
```

Comment:
This seems like a good idea? Are you planning to do it later in this pull request?